RecoTracker/LSTCore/src/alpaka/LSTESData.dev.cc

```diff
@@ -49,18 +49,18 @@ namespace {
                     std::shared_ptr<SDL::ModuleConnectionMap<SDL::Dev>> moduleConnectionMap) {
     // Module orientation information (DrDz or phi angles)
     auto endcap_geom =
-        get_absolute_path_after_check_file_exists(trackLooperDir() + "/data/OT800_IT615_pt0.8/endcap_orientation.bin");
+        get_absolute_path_after_check_file_exists(trackLooperDir() + "/data/OT800_IT615_pt0.6/endcap_orientation.bin");
```

Comment:
if the small efficiency loss is confirmed in a cmssw test with more events, should this part be aligned with the cut value? For `>= 0.8` use `pt0.8` and 0.6 for the cases below.

Reply:
To make the transition smoother, should I just add a on/off compile-time flag for the low pT setup alongside the runtime pT cut option? That way the low pT setup can be developed independently from the 0.8 pT cut setup, and we don't have this weird situation where part of the code uses the low pT setup (like the module maps or occupancies PR I'm making after this one or a t5 DNN trained on low pT events) and other parts use the current setup.